packages/mdc-fab/README.md

```diff
@@ -141,7 +141,8 @@ in all browsers. See [this Stackoverflow post](http://stackoverflow.com/posts/35
 
 CSS Class | Description
 --- | ---
-`mdc-fab--mini` | Makes the fab smaller (40 x 40 pixels)
+`mdc-fab--exited` | When this class is added, the FAB will shrink out of view (exit);<br>when it is removed, the FAB will grow into view (enter)
```

Comment:
mdc-fab-withdrawn??

drop the 'when this class is added'...I think thats implied with CSS classes

how about 'makes the FAB withdraw from view. When `mdc-fab--withdrawn` is removed, the FAB will return to view.

You don't have to mention grow/shrink, since I think thats an implementation detail

Reply:
I'll admit that in plain English, "withdrawn" sounds less weird than "exited", but I picked "exited" because "exit" is the specific word we use to refer to this motion, and it is a word that is meaningful to animators. WDYT?

Reply:
talked offline, --exited makes sense